cddl/query.cddl

```diff
@@ -1,6 +1,8 @@
 query = {
   &(artifact-type: 0) => artifact-type
   &(environment-selector: 1) => environment-selector-map
+  &(timestamp: 2) => tdate ; RFC3339 date
+  ? &(include-rims: 3) => bool .default false
```

Comment:
Can we call this `include-audit` rather than `include-rims`. Or at least choose a slightly more agnostic term than `rims`, because I'm worried that it might be implying more about the structure of the audit data than we can reasonably know. "Audit" might not be the best term, either, because that somewhat implies a whole process rather than just raw data. Maybe `include-audit-material` or `include-source-material` or something similarly evocative?

Reply:
OK.  I’d go with `include-source-material`.